client/scripts/controllers/server/staking.ts

```diff
@@ -0,0 +1,10 @@
+import app from 'state';
+import { get } from 'lib/util';
+
+class StakingController {
```

Comment:
Why are you creating a new controller for this, why not add it to the existing staking controller?

Reply:
We have this and so it should be used: https://github.com/hicommonwealth/commonwealth/blob/mir.staking-ui-events/client/scripts/controllers/chain/substrate/staking.ts

Reply:
The server folder has all the API calls to the backend, so I have to create a separate controller in server. I thought I should keep it the same way.

Reply:
I see, yea I think it would be best to move it to the other one for now.

Reply:
Optionally, we could put this in a ChainEvents controller. Though, I don't think we need a standalone staking controller for the server in this form, since it is so sparsely populated and likely will remain that way.

Reply:
Yes. I have moved the API call to ChainEntityController.